plugin/evm/block_verification.go

```diff
@@ -180,18 +180,10 @@ func (v blockValidator) SyntacticVerify(b *Block, rules params.Rules) error {
 
 	// If we are in ApricotPhase4, ensure that ExtDataGasUsed is populated correctly.
 	if rules.IsApricotPhase4 {
-		// Make sure ExtDataGasUsed is not nil and correct
-		if ethHeader.ExtDataGasUsed == nil {
```

Comment:
`utils.BigLessOrEqualUint64` and `utils.BigEqualUint64` both handle `nil` values internally.